mini-apps/acceleration-vlasiator/map_test.cpp

```diff
@@ -5,6 +5,10 @@
 #include "vlasovsolver/cpu_1d_ppm.hpp"
 #include "vlasovsolver/cpu_1d_plm.hpp"
 
+#define cell_pop_threshold 1.e-15
+// In vlasiator, called via spatial_cell->getVelocityBlockMinValue(popID)
```

Comment:
This comment is confusing, because it seems to imply that this is a parameter that can be changed at runtime, or by calling that function?

Maybe it should rather be something like "The slope limiters are numerically inaccurate for low values, thus the phase space densities are scaled by this divisor".

(This also makes me wonder, if it makes any sense to still call it "threshold" here, since it's really only used for scaling)